docs/docs-content/clusters/cluster-groups/create-cluster-group.md

```diff
@@ -53,11 +62,6 @@ s
 
 :::
 
-:::info
-
- Palette does not offer support for host clusters of these types within a cluster group: edge clusters, virtual clusters, Private Cloud Gateway (PCG) clusters, and imported clusters with read-only access.
-
```

Comment:
Judging by the line numbers, he wants to repeat this info here that also appears in lines 15-20. Is that right?

Reply:
No, I moved the infobox content to the top, which is not in those lines you mentioned.